api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/CardClient.java

```diff
@@ -49,4 +51,21 @@ public CardResponse requestNewCard(CardRequest cardRequest) {
             throw new CardClientException(e, cardsUri);
         }
     }
+
+    public DepositFundsResponse depositFundsToCard(String cardAccountId, DepositFundsRequest depositRequest) {
+        String uri = String.format("%s/%s/deposit", cardsUri, cardAccountId);
+        try {
+            ResponseEntity<DepositFundsResponse> response = restTemplate.postForEntity(uri, depositRequest, DepositFundsResponse.class);
+            if (response.getStatusCode() != HttpStatus.OK) {
+                log.error("Expecting an OK response from the card service api, instead received {} with response body {}",
+                        response.getStatusCode().name(), response.getBody());
+                throw new CardClientException(response.getStatusCode());
+            }
+
+            return response.getBody();
+        } catch (RestClientException e) {
+            log.error("Exception caught trying to post to {}", uri);
```

Comment:
Are we getting much value from adding this catch block? It won't tell us much more than what's logged in the exception handler.